extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -587,7 +586,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	getNavigation(uri: vscode.Uri): Thenable<azdata.nb.NavigationResult> {
 		let result: azdata.nb.NavigationResult;
 		let notebook = this.currentBook?.getNotebook(uri.fsPath);
-		if (notebook) {
+		// checking viewId to verify that the book is opened in the Notebooks Viewlet
```

Comment:
FYI it's "view" in this case, not "viewlet". Viewlet's are the sidebar items - each tree in a viewlet is a separate view.